SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -298,6 +298,18 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(Config)
 	uint32 UdpClientDownstreamUpdateIntervalMS;
 
+	/** Explicit flush intervals instead of relying on a flush interval. Flush is triggered by events such as RPCs. */
```

Comment:
If `bWorkerFlushAfterRPC` isn't set, this won't flush after RPCs right? Perhaps just say it flushes at the end of a frame?